src/Umbraco.Web/UmbracoHelper.cs

```diff
@@ -601,37 +597,40 @@ public IEnumerable<IPublishedContent> ContentAtRoot()
             return ContentQuery.ContentAtRoot();
         }
 
-        private static bool ConvertIdObjectToInt(object id, out int intId)
+        /// <remarks>Had to change to internal for testing.</remarks>
+        internal static bool ConvertIdObjectToInt(object id, out int intId)
         {
-            var s = id as string;
-            if (s != null)
+            switch (id)
             {
-                return int.TryParse(s, out intId);
-            }
+                case string s:
+                    return int.TryParse(s, out intId);
 
-            if (id is int)
-            {
-                intId = (int) id;
-                return true;
+                case int i:
+                    intId = i;
+                    return true;
+
+                default:
+                    intId = default;
+                    return false;
             }
-            intId = default(int);
-            return false;
         }
 
-        private static bool ConvertIdObjectToGuid(object id, out Guid guidId)
+        /// <remarks>Had to change to internal for testing.</remarks>
```

Comment:
No need for this comment to be in source code, just leave a comment in Github on the PR next time.